### ISSUE TITLE
500 When Forking

### ISSUE THREAD

**Issue by jag3773** [unfoldingWord-dev/gogs#101]
When trying to fork the [en-uhg](https://git.door43.org/Door43/en-uhg) repo as user bellekell I receive a 500 error.  See attached screenshot.
![screen shot 2016-11-30 at 4 42 50 pm](https://cloud.githubusercontent.com/assets/107731/20776182/4e0a49c0-b71c-11e6-9286-66de45bb481a.png)

Note this happens on every repo I've tried to fork as this user.  Possibly relevant sanitized log data from the server:
```
Nov 30 23:45:15 git gogs: [Macaron] 2016-11-30 23:45:15: Completed /js/jquery.are-you-sure.js 404 Not Found in 7.826229ms
Nov 30 23:45:17 git gogs: [Macaron] 2016-11-30 23:45:17: Started POST /repo/fork/5047 for 127.0.0.1
Nov 30 23:45:17 git gogs: 2016/11/30 23:45:17 #033[1;31m[...routers/repo/pull.go:132 ForkPost()] [E] ForkPost: updateUser: e-mail has been used [email: XXXXXXXXXXXX]#033[0m
Nov 30 23:45:17 git gogs: 2016/11/30 23:45:17 #033[1;34m[D] Template: status/500#033[0m
Nov 30 23:45:17 git gogs: [Macaron] 2016-11-30 23:45:17: Completed /repo/fork/5047 500 Internal Server Error in 18.480572ms
```

Seems like the ForkPost() is trying to update the user's email address?  Is that possible?

See if this can be reproduced on latest code.  If it can, create new issue to fix the problem.  Otherwise, retest on production after #100 .

**Comment by richmahn** [comment]
You can't make a repo as user bellekell because that user and jesse_test both have an email address. This is a known issue that needs to be fixed. tS develoeprs were having problems with this.

If you change the email address for one of those users, it will then work.

**Comment by richmahn** [comment]
Closing this. If we need to fix this, we need to open a new issue.
